frontend/src/routes/_dashboard/create-project.lazy.tsx

```diff
@@ -28,11 +35,50 @@ function AuthedView() {
 }
 
 export function CreateProject() {
+  const navigate = useNavigate();
+
+  const { isPending, mutate, data } = useMutation({
+    mutationFn: async (input: RequestInput<"post", "/api/projects/">) => {
+      await apiClient.GET("/api/csrf/");
+      const csrfToken = getCookie("csrftoken");
+      const { error, data } = await apiClient.POST("/api/projects/", {
+        headers: {
+          "X-CSRFToken": csrfToken
+        },
+        body: input
+      });
+
+      if (error) return error;
+      if (data) {
+        navigate({ to: "/" });
+        return;
+      }
+    }
+  });
+
+  const errors = getFormErrorsFromResponseData(data);
+
   return (
     <main>
-      <Form.Root className="flex h-[60vh] flex-grow justify-center items-center">
+      <Form.Root
+        action={(formData) =>
+          mutate({
+            slug: slugify(formData.get("slug")!).toString().trim(),
```

Comment:
I like the idea of slugifying the name of the project, but it might cause confusion, as many project names can have different slugs, I know I said it in the issue, but I don't think it's a great idea anymore.

Also, do not use exclamation point here but `?`  as if the user removes the `slug` input, it will cause a big unrecoverable error.